src/test/fixtures/profiles/processed-profile.js

```diff
@@ -1381,9 +1385,9 @@ export function getProfileWithUnbalancedNativeAllocations() {
     { byteSize: 5, stack: Gjs },
     { byteSize: 7, stack: I },
     // Deallocations:
-    { byteSize: -11, stack: E },
-    { byteSize: -13, stack: Gjs },
-    { byteSize: -17, stack: I },
+    { byteSize: -11, stack: J },
+    { byteSize: -13, stack: K },
+    { byteSize: -17, stack: Gjs },
```

Comment:
I take care that the deallocation sites are different than the allocation sites, as it is usually the case in real profiles.

Reply:
Note: this change doesn't come from a bug, but I think this is more future proof. I did it because I did a similar change in the next commit.

Reply:
Praise: Nice choice!